lightning/src/util/test_utils.rs

```diff
@@ -172,41 +174,105 @@ impl events::MessageSendEventsProvider for TestChannelMessageHandler {
 	}
 }
 
+fn get_dummy_channel_announcement(short_chan_id: u64) -> msgs::ChannelAnnouncement {
```

Comment:
Test helpers move could have been commented in commit or at least splitted to ease review. That's okay AFAICT, they don't break anything.

Reply:
I just commented it in the commit.